scripts/create_registry_schema.py

```diff
@@ -341,14 +332,15 @@ def _Dependency(schema, has_production, production="production"):
         print(f"Could not grant access to {acct} on schema {schema}")
 
 # Create the tables
-# for SCHEMA in SCHEMA_LIST:
-_Dataset(schema)
-_DatasetAlias(schema)
-_Dependency(schema, db_connection.dialect != "sqlite",
-            production=prod_schema)
-_Execution(schema)
-_ExecutionAlias(schema)
-_Provenance(schema)
+for table_name in [
```

Comment:
A construction like this, where the code specifies table names, shouldn't be necessary.   There could be a routine `get_table_names()` which would look them up from `schema_data`

Reply:
Done